Formula/zeek.rb

```diff
@@ -19,6 +19,9 @@ class Zeek < Formula
   depends_on "caf"
   depends_on "geoip"
   depends_on "openssl@1.1"
+  uses_from_macos "flex"
```

Comment:
```suggestion

  uses_from_macos "flex"
```

Can we have a blank line between `depends_on` and `uses_from_macos` please? The difference in length of the two is weird. At some point I'll write the audit to enforce this...